src/bootlint.js

```diff
@@ -937,12 +959,7 @@ var LocationIndex = _location.LocationIndex;
             var elem = $(this);
             var urlAttr = (tagNameOf(this) === 'LINK') ? 'href' : 'src';
             var pathSegments = parseUrl(elem.attr(urlAttr)).pathname.split('/');
```

Comment:
Those 3 lines? While we can for the W013 and W015 that negates it's use in W005 (lintJquery). I had chosen to include less to allow it to be used in more places